backend/floop_data_15k.csv

```diff
@@ -0,0 +1,15618 @@
+Field1
```

Comment:
Remove this file and give instructions on how to retrieve the data from S3 instead

Reply:
I did it.